Tools/Postgres/README.md

```diff
@@ -150,4 +150,33 @@ psql -h localhost -p 5432 -U postgres -c 'drop database gateway;'
 psql -h localhost -p 5432 -U postgres -c 'create database gateway;'
 ```
 
-4) Run Migrations Scripts
\ No newline at end of file
+4) Run Migrations Scripts
+
+## Edit Patroni configuration using Rest API
+
+1) Connect the terminal to any of the patroni pods running using remote shell:
+
+``` bash
+oc rsh patroni-postgres-1
+```
+
+2) Update config (e.g. postgresql.parameters.max_connections to 100):
+
+``` bash
+curl -s -XPATCH -d '{"postgresql":{"parameters":{"max_connections":100}}}' http://localhost:8008/config | jq .
```

Comment:
Does this hold true when a container restart occurs?

Reply:
yes!